pontoon/base/views.py

```diff
@@ -1087,7 +1085,6 @@ def transvision(request, repo, title):
 
             return HttpResponse(json.dumps({
```

Comment:
I'm with @jotes on this one, this is just visual consistency, which isn't really all that helpful.